pyproject.toml

```diff
@@ -120,7 +115,7 @@ profile = "black"
 
 [tool.mypy]
 # Sync Python version with Dockerfile
-python_version = "3.10.9"
```

Comment:
This config option only accepts specifying Python with a major and minor version -- it does not accept adding the patch version